Missing command for applying changes after dev:query-log command (#9448)
magento/devdocs: src/guides/v2.3/config-guide/cli/logging.md

   ```bash
   bin/magento dev:query-log:disable
   ```
   

> 2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` sets settings into app/etc/env.php file:

Reviewer: ```suggestion
2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` adds settings to the `app/etc/env.php` file:
```
```suggestion
2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` sets settings into app/etc/env.php file:
```

> 2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` sets settings into app/etc/env.php file:

Reviewer: ```suggestion
2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` adds settings to the  `app/etc/env.php` file:
```
```suggestion
2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` sets settings into app/etc/env.php file:
```

> 2. In the [production mode](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show) run the command for [import configuration](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the command `bin/magento dev:query-log:disable` adds settings to the `app/etc/env.php` file:

Reviewer 2: ```suggestion
1. In the [production mode]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html#config-mode-show), run the command for [import configuration]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-config-mgmt-import.html), because the `bin/magento dev:query-log:disable` command adds settings to the `app/etc/env.php` file:
```

  
  ```bash
  bin/magento app:config:import
  ```

> 3. Flush the cache.

Reviewer 2: For markdown, only use "1" to indicate a numbered list item.

```suggestion
1. Flush the cache.
```
